pybeautifyjson.fish

```diff
@@ -1,5 +1,7 @@
-# beautify json string
-# use : pybeautifyjson '{"foo": "lorem", "bar": "ipsum"}'
-function pybeautifyjson
-  echo $argv | _python -mjson.tool
+function pybeautifyjson -d "Beautify a JSON string"
```

Comment:
my local version of this (which is essentially identical, despite being independently invented ages ago) is called `jsonpp` for "json pretty print",